.buildkite/scripts/build.sh

```diff
@@ -16,7 +16,7 @@ docker run \
        --volume "$(pwd)"/dist:/src/dist \
        --rm \
        "${builder_container}" \
-       pyinstaller --onefile --add-data cogctl/GITSHA:. bin/cogctl
+       pyinstaller --onefile --add-data cogctl/GITSHA:. --add-data cogctl/GITTAG:. bin/cogctl
```

Comment:
If we're using the makefile for building, this should be changed there, and this script needs to be changed. The binary would be built inside the image built on line 9 above; we'd need to replace this present `docker run` command with something that would extract the binary from the image so we could upload it using the Buildkite agent.